### ISSUE TITLE
Table creation with gretl function 

### ISSUE THREAD

**Issue by sagirumati** [HelioGuilherme66/gretl#1]
I don't know if I can create a table and fill it in with values I want using gretl built in function. Otherwise, I would like to suggest that a function be created. I also want to be able to modify (e.g extract some rows or columns only) equation table  before exporting to csv or other available format, also using some built-in gretl functions. 

Thank you

**Comment by henrique-andrade** [comment]
> I don't know if I can create a table and fill it in with values I want using gretl built in function (...)

What kind of table are you talking? Maybe a matrix?

**Comment by sagirumati** [comment]
Yes something like matrix, but I want an object like dataframe in R.
I created an R package gretlR https://cran.r-project.org/web/packages/gretlR/index.htm , which runs gretl codes in R Markdown document.
I'm currently facing two problems,

1) I can't export some part of the gretl model table. For example I want to export row 1 to 4 only of the model table output using gretl commands. Or for example, I want create a table, then fill it in with headers and row names then export it in csv format using gretl command
2) I can't remove the tex comment from the model equation output before exporting it tex. Because I use R Markdown for document writing, so I export the model equation output in Rmd format, then include it using some command. Unfortunately, R Markdown does not read the tex comments as comment, it writes everything including the tex comment in the final output. This means my document cannot be dynamic as I have to manually remove the tex comments before knitting my document

Thank you for your effort

**Comment by henrique-andrade** [comment]
Dear Sagiru,

I think I can help you. Take a look at following code I just wrote for you (using **Gretl**). But I recommend you to join the Gretl development **[mailing list](https://gretlml.univpm.it/postorius/lists/gretl-devel.gretlml.univpm.it/)**. The rest of the team will love to see your project. In my opinion it's an awesome project :)

# Slicing Gretl's model table

## Step 1: Estimation of OLS models and inclusion in model table
```
set workdir "/home/henrique/Documentos"

open australia.gdt

"model 1" <- ols le const lpus lpau
modeltab add

"model 2" <- ols le const lpus
modeltab add

"model 3" <- ols le const lpau
modeltab add
```

## Step 2: Storing the model table on a file called `rawTable.txt`
```
modeltab --output="rawTable.txt"
```

## Step 3: Reading back the saved txt file
```
string rawTable = readfile("rawTable.txt")
```

## Step 4: Creating an empty array to store each line of the table separately
```
strings tableLines = array() 
```

## Step 5: Reading each line of the string `rawTable` and storing in the array `tableLines`
```
string line = ""

loop while getline(rawTable, line)
    tableLines = tableLines + line
endloop
```

## Step 6: Selecting `n` lines of the `tableLines` array
```
scalar n = 10
strings modifiedTable = tableLines[1:n]
```

To check if the selection is correct you can execute:
```
loop for i = 1..nelem(modifiedTable)
    eval modifiedTable[i]
endloop
```

## Step 7: Exporting the results to a new file called `modifiedTable.txt`
```
outfile modifiedTable.txt --quiet
    loop for i = 1..nelem(modifiedTable) --quiet
        eval modifiedTable[i]
    endloop
end outfile
```

**Comment by HelioGuilherme66** [comment]
@henrique-andrade @sagirumati Hello! Sorry for only now seeing this, somehow I did not get notification or missed it.

How is this going on? Did you adverstise the project on the mailing-list?
